ChangeLog

```diff
@@ -1,3 +1,8 @@
+2025-04-08  Charlie Gao  <charlie.gao@shikokuchuo.net>
+
+	* inst/include/asio*: Upgraded to Asio 1.28.2
+	* inst/include/asio/detail/config.cpp: Ensure ASIO_HAS_SNPRINTF is defined
```

Comment:
Thank you -- we could add the patch file here, but I canm also sneak that in when I do my iteration.